brvs-rules/src/main/java/iroha/validation/rules/impl/billing/BillingInfo.java

```diff
@@ -133,7 +133,7 @@ public boolean equals(Object other) {
     if (other == this) {
       return true;
     }
-    if (!this.getClass().isAssignableFrom(other.getClass())) {
+    if (other == null || !this.getClass().isAssignableFrom(other.getClass())) {
```

Comment:
I'd rather put this check at the top of the method.

Reply:
Okay